ui/apps/platform/src/services/DeploymentsService.ts

```diff
@@ -13,8 +13,8 @@ import { getRequestQueryStringForSearchFilter } from 'utils/searchUtils';
 import { CancellableRequest, makeCancellableAxiosRequest } from './cancellationUtils';
 import axios from './instance';
 
-const deploymentsUrl = '/v1/deploymentswithprocessinfo';
-const deploymentByIdUrl = '/v1/deployments';
+const deploymentUrl = '/v1/deployments';
```

Comment:
Distinguishing variable names for endpoints in this way is very helpful.

For the names to be parallel, what do you think of plural `deploymentsUrl`